backend/file/file.go

```diff
@@ -37,6 +41,10 @@ func NewBackend(path string) *Backend {
 func (b *Backend) Unmarshal(ctx context.Context, to interface{}) error {
 	f, err := os.Open(b.path)
 	if err != nil {
+		if b.optional {
+			fmt.Printf("failed to open file at path \"%s\": %s\n", b.path, err.Error())
```

Comment:
:warning: Never ever print in a library, log instead

Reply:
Good catch! I will add a basic logger for the whole library.

Reply:
Done in 4750396.